### PR TITLE
Set default network driver for APIv2 networks

### PR DESCRIPTION
**Pull request by baude**
Recent changes in networking require that the cni network driver be set.
If the user provides no driver, we set the driver to the
defaultnetworkdriver which currently is "bridge".

Fixes: #8294

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: test/apiv2/35-networks.at [hunk 1]
```diff
@@ -8,7 +8,7 @@ t GET networks/non-existing-network 404 \
 
 # FIXME FIXME FIXME: failing in CI. Deferring to someone else to fix later.
```
Comment (mheon) [inline test/apiv2/35-networks.at:9]: Remove comment

File: test/apiv2/35-networks.at [hunk 2]
```diff
@@ -6,9 +6,8 @@
 t GET networks/non-existing-network 404 \
   .cause='network not found'
 
-# FIXME FIXME FIXME: failing in CI. Deferring to someone else to fix later.
 #if root; then
-if false; then
+if root; then
```
Comment (Luap99) [inline test/apiv2/35-networks.at:10]: This check can be removed. We now support rootless networking. I tried running this as user and it worked.

**Comment by Luap99** [comment]
LGTM

**Comment by edsantiago** [comment]
LGTM, and the APIv2 test is passing, yay! Thank you!

**Comment by baude** [comment]
merge me

**Comment by edsantiago** [comment]
/lgtm

Thanks, @baude

**Comment by rhatdan** [comment]
Currently we don't support a default network for containers.  Should this be added to containers.conf?

**Comment by mheon** [comment]
We should already have one for CNI, at least?

**Comment by mheon** [comment]
```
       default_network="podman"

       The network name of the default CNI network to attach pods to.
```

**Comment by rhatdan** [comment]
Great.

**Pull request merged**